.travis.yml

```diff
@@ -1,12 +1,19 @@
 language: ruby
 cache: bundler
+dist: trusty
 sudo: false
-rvm: 2.4.1
+
+rvm:
+  - 2.3
```

Comment:
Why are we testing 2.3 and 2.3.4. It looks like when you ask for 2.3, that Travis creates an alias to 2.3.4:
```
rvm alias create 2.3 ruby-2.3.4
```